app/Http/Controllers/StatementController.php

```diff
@@ -62,9 +62,9 @@ public function index(Request $request): View|Factory|Application
         $total = $setup['total'];
 
         $similarity_results = null;
-        if ($request->get('s')) {
-            $similarity_results = $this->drive_in_service->getSimilarityWords($request->get('s'));
-        }
+        // if ($request->get('s')) {
```

Comment:
We no longer show these or use them. I keep them because I think no one has noticed.